### PR TITLE
hide VerifyAll

### PR DESCRIPTION
**Pull request by xutongye**
## Description
We need one and only one global instance of `VerifyAll`, and use this instance everywhere needed. 

This PR tries to "hide" the class `VerifyAll`, and only expose a global instance of it, named `VERIFYALL`. The way I "hide" `VerifyAll` is to define it inside a function, make it a local class, and hard for users to import and use it directly.

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: src/syft/core/common/group.py [hunk 1]
```diff
@@ -32,15 +32,21 @@ def proto2object(proto: VerifyKey_PB) -> VerifyKey:
...
-        return VerifyAllWrapper_PB(all=Empty_PB())
-
-    @staticmethod
-    def _proto2object(proto: VerifyAllWrapper_PB) -> Type:  # type: ignore
-        return VerifyAll
-
-    @staticmethod
-    def get_protobuf_schema() -> GeneratedProtocolMessageType:
-        return VerifyAllWrapper_PB
+def _create_VERIFYALL() -> Any:
```
Comment (tudorcebere) [inline src/syft/core/common/group.py:35]: Why not make it a Singleton just to be sure instead of hiding it in a function?

CC: @madhavajay

Comment (xutongye) [reply src/syft/core/common/group.py:35]: @madhavajay said that we don't want users call like `isinstance(some_key, VerifyAll)`, but want them to use `some_key is VERIFYALL`.

Comment (madhavajay) [reply src/syft/core/common/group.py:35]: I think we can make this a singleton as well just make sure its not possible to get the type and then create a different copy. So there are two things here:
✅ 1) hide the class from import
☑️ 2) make it a singleton so you end up with if if you try to make another one from its type constructor

File: src/syft/util.py [hunk 2]
```diff
@@ -104,6 +104,11 @@ def index_syft_by_module_name(fully_qualified_name: str) -> object:
 
     """
     attr_list = fully_qualified_name.split(".")
+
+    # we deal with VerifyAll differently, because we don't it be imported and used by users
+    if attr_list[-1] == "VerifyAll":
+        return type(syft.core.common.group.VERIFYALL)
```
Comment (madhavajay) [inline src/syft/util.py:110]: If it was a singleton perhaps this would let new construction match?

**Review by tudorcebere — APPROVED** [review]

**Pull request merged**